src/services/fileStorage/utils/filePathHelper.js

```diff
@@ -1,9 +1,32 @@
 const removeLeadingSlash = (path) => (path[0] === '/' ? path.substring(1) : path);
 
-const generateFileNameSuffix = (fileName) => {
+const s3FileNameFilterExpression = /[^0-9A-Za-z\-_.]+/g;
```

Comment:
Can you please add the not added chars to the test to see if it is similar to 
fileName.replace(/[';:=#*+\[\]~<{\\()}>§$%&|^£@±?!"`□\s]/g, '-')

Reply:
added in tests, while this is no user-dependency and only a technical detail... in https://github.com/hpi-schul-cloud/schulcloud-server/pull/2875/files#diff-c07fd68a1eba283b454e826b7f6ae0eca1c653478f055f01bea9cb9e503db5e6R51